src/styles/index.scss

```diff
@@ -1,3 +1,236 @@
+$primary-color: #00acdc;
+$font-color: #060b35;
+$light-grey: #616070;
+$font-family: 'Roboto', sans-serif;
+
+html {
+  margin: 0;
+  min-width: 1024px;
+  font-family: $font-family;
+  font-size: 12px;
+  line-height: 14.06px;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: end;
+  gap: 243px;
+  height: 60px;
+  background-color: #fff;
+  font-family: $font-family;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  padding-right: 50px;
+}
+
+.logo {
+  display: flex;
+  position: absolute;
+  top: 10px;
+  left: 50px;
+  margin: 0;
+}
+
+.nav__list {
+  display: flex;
+  gap: 20px;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  color: #060b35;
+  cursor: pointer;
+  text-transform: uppercase;
+  transition: color 0.3s;
+  position: relative;
+  line-height: 60px;
+  display: block;
+  height: 60px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.nav__link:hover {
+  color: $primary-color;
+}
+
+.is-active {
+  color: $primary-color;
+  position: relative;
+  text-decoration: none;
+}
+
+.is-active::after {
+  content: '';
+  background-color: $primary-color;
+  border-radius: 8px;
+  width: 100%;
+  height: 4px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+  padding: 50px 40px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+* {
+  box-sizing: border-box;
+}
+
+h2,
+p {
+  margin: 0;
+}
+
+.card {
+  border-radius: 5px;
+  width: 200px;
+  margin: 0;
+  padding: 32px 16px 16px;
+  font-family: $font-family;
+  border: 1px solid #f3f3f3;
+  transition:
+    transform 0.3s ease,
+    color 0.3s ease;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+  display: block;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: $font-color;
+  margin-bottom: 4px;
+  transition: color 0.3s ease;
+}
```

Comment:
Move these styles to the separate card file styles styles, and use scss nesting everywhere